src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -25,10 +25,10 @@
 
     public static final Person ALICE = new PersonBuilder().withName("Alice Pauline")
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
-            .withPhone("94351253").withRemark("She likes aardvarks.")
+            .withPhone("94351253")
             .withTags("friends").build();
     public static final Person BENSON = new PersonBuilder().withName("Benson Meier")
-            .withAddress("311, Clementi Ave 2, #02-25").withRemark("He can't take beer!")
+            .withAddress("311, Clementi Ave 2, #02-25")
             .withEmail("johnd@example.com").withPhone("98765432")
             .withTags("owesMoney", "friends").build();
     public static final Person CARL = new PersonBuilder().withName("Carl Kurz").withPhone("95352563")
```

Comment:
Not sure if the PR should include the tutorial commits.
Perhaps need to work on a different branch and PR separately.

Reply:
I think it might be possible that the tutorial commits were not done in a separate branch or maybe it was merged to the master branch on Aditi's local and forked repo. 

I think the commits might get merged into the master branch on the team repo if we merge this pull request.
However, I am not sure why the files changed do not reflect these new additions from the tutorials.